packages/cli-e2e/entrypoints/utils/common.js

```diff
@@ -5,7 +5,7 @@ const {existsSync, mkdirSync, writeFileSync} = require('fs');
 const DOCKER_IMAGE_NAME = 'coveo-cli-e2e-image';
 const composeProjectName = 'coveo-cli-e2e';
 const DOCKER_CONTAINER_NAME = 'coveo-cli-e2e-container';
-const repoHostPath = resolve(__dirname, ...new Array(3).fill('..'));
+const repoHostPath = resolve(__dirname, ...new Array(4).fill('..'));
```

Comment:
What does that 3->4 represent ? What does it reference ?

And second question, can we retrieve it "correctly" in another manner so that it does not happen ?

Reply:
> What does that 3->4 represent ? What does it reference ?
the `Array(n)` represent the number of `..` to add to the path resolving (i.e. how many time do we have to go one folder higher in the hierarchy). This is more readable IMO than `'..','..'` four times.

> And second question, can we retrieve it "correctly" in another manner so that it does not happen ?
No, we cannot afaik. It's pathing, not an import of any kind. If the files moves, we have to update the path. There's no 'automagic' correction possible being at the IDE level or anything afaik.